### ISSUE TITLE
Gradient disappears in CIFAR10

### ISSUE THREAD

**Issue by huanranchen** [facebookresearch/ConvNeXt#98]
Hi~ When training on cifar10, I discovered that even in the first iteration, the gradient of my model disappears. But if I fine tune pretrained ConvNeXt on cifar10, the gradient is normal.
<img width="960" alt="56f77cefaa2b26f8712cd9e98e4443b" src="https://user-images.githubusercontent.com/28839782/166642028-55bfdaf2-c27f-4d34-8181-8387e2750a9b.png">
I use convnext in pytorch, with learning_rate 4e-3, AdamW optimizer, batch_size 64.
```
class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()
        self.model = models.convnext_tiny(pretrained=False)

        self.classifier = nn.Sequential(
            nn.Flatten(start_dim=1, end_dim=-1),
            nn.LayerNorm([768], eps=1e-6, elementwise_affine=True),
            nn.Linear(768, 60)
        )

    def forward(self, x):
        x=self.model.features(x)
        x = self.model.avgpool(x)
        x = self.classifier(x)

        return x
```

This problem is not only in cifar10, but also in NICO dataset where the input image size is 224x224. I tried to resize the input image to 448x448, but the gradient is still abnormal.

So, why the gradient will disappear~

**Comment by huanranchen** [comment]
I have solved this problem.... Init is important for convnext...

    def _init_weights(self, m):
        if isinstance(m, (nn.Conv2d, nn.Linear)):
            trunc_normal_(m.weight, std=.2)
            nn.init.constant_(m.bias, 0)

**Comment by huanranchen** [comment]
> I have solved this problem.... Init is important for convnext...
> 
> ```
> def _init_weights(self, m):
>     if isinstance(m, (nn.Conv2d, nn.Linear)):
>         trunc_normal_(m.weight, std=.2)
>         nn.init.constant_(m.bias, 0)
> ```

But, this can only solve the problem on cifar 10....... After one epoch training on NICO dataset, the gradient disappeared...

**Comment by huanranchen** [comment]
就是，无论是在CIFAR10上还是NICO上都会出现梯度消失，并且是否使用预训练模型都一样，只有在CIFAR10上用0.2 trunc normal才能解决这个问题...

**Comment by huanranchen** [comment]
I don't know how I solved this problem. I use same code, same environment, and managed to train on cifar10 today.....
